.github/workflows/shared-branch-push-java.yml

```diff
@@ -62,7 +62,6 @@ jobs:
           mavenS3SecretAccessKey: ${{ secrets.TEMP_MAVEN_S3_SECRET_ACCESS_KEY }}
```

Comment:
Can't remove it from inputs in this file yet as that would break services still providing it.